lua/care/types/source.lua

```diff
@@ -28,11 +28,11 @@
 --- An example for this could be `.`, `\\` and `/` when working with paths.
 ---
 --- ```lua
---- function my_source.get_trigger_characters()
+--- function my_source:get_trigger_characters(self)
```

Comment:
```suggestion
--- function my_source:get_trigger_characters()
```

either you have the `self` parameter or the `:`  but not both

Reply:
Whoops you're right, fixed it.